### ISSUE TITLE
Request headers should not have more than 3 digits in the `q=` value

### ISSUE THREAD

**Issue by DylanVanAssche** [kianschmalenbach/rdf-browser#32]
# Description

I have this plugin installed and works great with most things.
However, it breaks the Solid Community Server IDP:

![image](https://user-images.githubusercontent.com/4999159/146522762-5121666c-bab9-4343-9966-fba47c042f7c.png)

After some internal discussion, the cause seems to be this plugin as it generates floating point values with more than 3 digits.
This is against the RFC7231 specification: https://datatracker.ietf.org/doc/html/rfc7231#section-5.3.1 which explicitly forbids it as:

> A sender of qvalue MUST NOT generate more than three digits after the decimal point. 

Disabling the plugin makes it work again.

**Request headers**
```
Accept: application/ld+json;q=1,application/n-quads;q=1,application/n-triples;q=1,application/rdf+xml;q=1,application/trig;q=1,text/turtle;q=1,text/n3;q=1,application/ld+json;q=0.95,application/n-quads;q=0.95,application/n-triples;q=0.95,application/rdf+xml;q=0.95,application/trig;q=0.95,text/turtle;q=0.95,text/n3;q=0.95,text/html;q=0.8999999999999999,application/xhtml+xml;q=0.8999999999999999,application/xml;q=0.7999999999999999,image/avif;q=0.8999999999999999,image/webp;q=0.8999999999999999,*/*;q=0.7
Accept-Language: en;q=0.9, de;q=0.8, *;q=0.5
```

# How to reproduce

- Enable the plugin
- Check the request headers
- Requests headers contain `q` with floating points with more than 3 digits after the decimal point

# Environment

- Fedora Workstation 35
- Firefox 95.0
- RDF Browser 1.2.2

**Comment by argahsuknesib** [comment]
I faced this issue as well.

**Comment by kianschmalenbach** [comment]
Fixed in version 1.2.3.
